desktop-widgets/btdeviceselectiondialog.cpp

```diff
@@ -286,10 +296,25 @@ void BtDeviceSelectionDialog::hostModeStateChanged(QBluetoothLocalDevice::HostMo
 #endif
 }
 
+int BtDeviceSelectionDialog::getDevicePriority(bool connectable, const QBluetoothDeviceInfo &remoteDeviceInfo)
```

Comment:
do these priorities have a meaning? Should they be constants?

Reply:
Well, they are priorities, i.e. they define an absolute order. See commit message. Of course, we want to extend that by heuristics taking into account the selected dive computer - or vice versa: select the correct dive computer based on the name of the BT device.